packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -384,12 +388,14 @@ export async function downloadDocsAndUpsertFHIR({
   documents,
   forceDownload = false,
   ignoreDocRefOnFHIRServer = false,
+  requestId,
 }: {
   patient: Patient;
   facilityId: string;
   documents: Document[];
   forceDownload?: boolean;
   ignoreDocRefOnFHIRServer?: boolean;
+  requestId: string;
```

Comment:
nit: there are some places in this func where we throw errors (ie ` capture.error`), so it could be useful to include the requestId there as well - note that this generally applies throughout the PR